src/Symfony/Bundle/SecurityBundle/Debug/TraceableFirewallListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\Debug;
+
+use Symfony\Bundle\SecurityBundle\EventListener\FirewallListener;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+
+/**
+ * Firewall collecting called listeners.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ *
+ * @final since version 3.4
```

Comment:
Why are you using the tag here instead of marking the class as being final directly?